nri_device_injector/nri_device_injector.go

```diff
@@ -99,18 +99,21 @@ func (p *plugin) CreateContainer(_ context.Context, pod *api.PodSandbox, contain
 	l.Info("Started CreateContainer")
 	devices, err = getDevices(ctrName, pod.Annotations)
 	if err != nil {
+		l.Warnf("Failed to get device from pod annotation: %v", err)
 		return nil, nil, err
 	}
 	adjust := &api.ContainerAdjustment{}
 
 	if len(devices) == 0 {
 		l.Debug("No devices annotated...")
+		l.Info("Finished CreateContainer")
```

Comment:
If we want this kind of log, maybe put it at the top of the function with a `defer`?

Reply:
Done, thanks for the suggestion!